cmd/image-loader/main.go

```diff
@@ -89,7 +90,7 @@ func main() {
 	flag.StringVar(&o.versionFilter, "version-filter", "", "Version constraint which can be used to filter for specific versions")
 	flag.StringVar(&o.registry, "registry", "", "Address of the registry to push to, for example localhost:5000")
 	flag.BoolVar(&o.dryRun, "dry-run", false, "Only print the names of found images")
-	flag.StringVar(&o.addonsPath, "addons-path", "", "Path to the folder containing the addons")
+	flag.StringVar(&o.addonsPath, "addons-path", "", "Path to a directory containing the KKP addons, if not given, falls back to the Docker image configured in the KubermaticConfiguration")
```

Comment:
Should not be better to have an `addons-image` flag that defaults to the addon image for this version instead of taking it from the `KubernetesConfiguration`? This way would work even when `version-file` is given.